tests/tests/BuilderTest.php

```diff
@@ -168,45 +181,52 @@ public function testReducePaths($reducedPaths, $commonPath, $paths)
 
     public function reducePathsProvider()
     {
-        return [
-            [
+        $s = \DIRECTORY_SEPARATOR;
+
+        yield [
+            [],
+            ".",
+            []
+        ];
+
+        $prefixes = ["C:$s", "$s"];
+
+        foreach($prefixes as $p){
+            yield [
                 [
-                    'Money.php'    => [],
-                    'MoneyBag.php' => []
+                    "Money.php" => []
                 ],
-                '/home/sb/Money',
+                "{$p}home{$s}sb{$s}Money{$s}",
                 [
-                    '/home/sb/Money/Money.php'    => [],
-                    '/home/sb/Money/MoneyBag.php' => []
+                    "{$p}home{$s}sb{$s}Money{$s}Money.php" => []
                 ]
-            ],
-            [
+            ];
+
+            yield [
                 [
-                    'Money.php' => []
+                    "Money.php"    => [],
+                    "MoneyBag.php" => []
                 ],
-                '/home/sb/Money/',
+                "{$p}home{$s}sb{$s}Money",
                 [
-                    '/home/sb/Money/Money.php' => []
+                    "{$p}home{$s}sb{$s}Money{$s}Money.php"    => [],
+                    "{$p}home{$s}sb{$s}Money{$s}MoneyBag.php" => []
                 ]
-            ],
-            [
-                [],
-                '.',
-                []
-            ],
-            [
+            ];
+
+            yield [
                 [
-                    'Money.php'          => [],
-                    'MoneyBag.php'       => [],
-                    'Cash.phar/Cash.php' => [],
+                    "Money.php"          => [],
```

Comment:
Please dont replace `'` with `"` here.

Reply:
Only that? I thought to maintain the style consistent in that method. The double quotes are not needed in many other places.